src/components/ha-cover-controls.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+import { HassEntity } from "home-assistant-js-websocket";
+
+import { UNAVAILABLE } from "../data/entity";
+import { HomeAssistant } from "../types";
+import CoverEntity from "../util/cover-model";
+
+import "./ha-icon-button";
+
+@customElement("ha-cover-controls")
+class HaCoverControls extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @internalProperty() private _stateObj!: HassEntity;
+
+  private _entityObj?: CoverEntity;
+
+  public set stateObj(stateObj: HassEntity) {
+    this._entityObj = new CoverEntity(this.hass, stateObj);
+    this._stateObj = stateObj;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._entityObj) {
+      return html``;
+    }
+
+    return html`
+      <div class="state">
+        <ha-icon-button
+          class=${classMap({
+            invisible: !this._entityObj.supportsOpen,
+          })}
+          label=${this.hass.localize("ui.dialogs.more_info_control.open_cover")}
```

Comment:
Am I missing something or this not supposed to produce a tooltip? I thought there was some big overhaul to support this or something?

Reply:
```suggestion
          .label=${this.hass.localize("ui.dialogs.more_info_control.open_cover")}
```

Maybe?

Reply:
```suggestion
          .title=${this.hass.localize("ui.dialogs.more_info_control.open_cover")}
```

Or is it title? idk honestly

Reply:
should be `.label` but it is just for an aria label, no tooltip.